src/websocket.js

```diff
@@ -0,0 +1,35 @@
+
+import { Client } from '@stomp/stompjs';
+import websocket from 'websocket';
+
+Object.assign(global, { WebSocket: websocket.w3cwebsocket })
+
+const client = new Client({
+  brokerURL: 'ws://localhost:8080/mywebsockets',
```

Comment:
cna you make this an or statement with the deployed endpoint as well?

Reply:
I think you can only have one brokerURL per client and we'll have to just change this to the deployed one later or have a seperate client?